src/main/java/org/kuali/kfs/pdp/businessobject/ACHBank.java

```diff
@@ -252,4 +256,14 @@ public String getBankCountryCode() {
     public void setBankCountryCode(String bankCountryCode) {
         this.bankCountryCode = bankCountryCode;
     }
+
+    public void setBankState(State bankState) {
+        LOG.info("setBankState, setting bank state to: " + ToStringBuilder.reflectionToString(bankState));
```

Comment:
Should these logging statements be set to debug level instead of info level?

Also, if you switch over to using lang3 ToStringBuilder, I think that version of it does not allow null values for "reflectionToString".

Reply:
Good point about making these debug log statements.  I shall change.  And yes, you are correct ToStringBuilder errors on null with the lang3 version